packages/web-components/src/components/va-search-input/va-search-input.scss

```diff
@@ -81,11 +81,11 @@ button.usa-search__clear-input:not([disabled]):focus {
   outline-offset: -4px;
 }
 
```

Comment:
The attributes were being set without a value, which is expected. This change makes the close button alignment for when the attributes are present or have a value